backend/environment/env.graph-service.template

```diff
@@ -0,0 +1,85 @@
+# Copy this file to "<project_root>/.env.graph-service", and then tweak values for local development
+# Values in this file will override the same-named environnent variables in `.env.common.docker` for the graph-service
+
+# Max number of jobs allowed on the 'graphUpdateQueue' before
+# blockchain scan will be paused to allow queue to drain
+QUEUE_HIGH_WATER=1000
+
+# Number of seconds to retain pending graph updates in the Redis cache to avoid redundant fetches from the chain
+DEBOUNCE_SECONDS=10
+
+# Maximum amount of provider capacity this app is allowed to use (per epoch)
+#     type: 'percentage' | 'amount'
+#     value: number (may be percentage, ie '80', or absolute amount of capacity)
+CAPACITY_LIMIT='{"type":"percentage", "value":80}'
+
+# Graph environment type. This can be 'Dev' or 'Rococo', 'TestnetPaseo', or 'Mainnet'.
+GRAPH_ENVIRONMENT_TYPE=Dev
```

Comment:
We should likely default everything to `TestnetPaseo` and use testnet paseo Frequency urls as default.

Reply:
One of the things I want to do coming up is eliminate the need to have graph-sdk hard-code the schema IDs and simply let the client specify them, since they can now be queried from the chain.  Then we don't need to specify the chain type.